### DISCUSSION TITLE
Embed Web3Auth Solution into Squarespace

### DISCUSSION THREAD

**Discussion by anarcocrypto** [Web3Auth/Web3Auth#622] [Help]
Hi! I'm Carlos, talking from Brazil
How are you all doing?
It's nice to be here. I'm not a real dev but I'm working on a project at my company and I need to make a simple integration with the phantom wallet on my website. Just a button to connect the user's wallet so we get his address. I've tried some ways like using Visual Studio Code and Codepen, but the first one only will work if I keep the VSC open and the second one is not working (I've used this video as reference: https://www.youtube.com/watch?v=OXWT74OPb2o). Here we use Squarespace to manage our website and it allows us to embed a link. So I just need some platform to create this button and get the information because I will embed it to our application form anyway using.

So I've found this Web3Auth solution and it would be great to put it on my Squarespace webpage to connect with those who fill our forms. 

Any thoughts? 

Thanks!

**Comment by shahbaz17** [comment]
Hey @anarcocrypto 

Please take a look https://web3auth.io/docs/integration-builder?lang=react&chain=sol&customAuthentication=no&whitelabel=no&mfaParams=no&usingEmailPasswordless=no&rnWorkflowMode=expo&evmFramework=web3. 

This will guide you on how to add web3auth.
